src/js/task.js

```diff
@@ -102,16 +102,16 @@ export default class Todolist {
     this.taskDtata.forEach((element, index) => {
       element.index = index;
     });
-    localStorage.setItem('TODOLISTDB', JSON.stringify(this.taskDtata));
-    this.display();
+    // localStorage.setItem('TODOLISTDB', JSON.stringify(this.taskDtata));
+    // this.display();
```

Comment:
As requested in the project requirements:
![image](https://user-images.githubusercontent.com/42799579/207926403-ba32a1b2-336c-4ede-bc40-c9970628544c.png)

Currently, your project doesn't contain any mock for the localStorage functions. Please make sure to create a mock and test them 😉. This will save you time when updating your project functionality and make it easier to scale 😄.

> Personal advice: I know that testing is difficult and takes a lot of time to learn, but from my experience, I highly suggest that you try to understand these concepts as soon as you can. It is extremely common in real development jobs to request testing for all implementations, without mentioning [TDD](https://www.devmedia.com.br/test-driven-development-tdd-simples-e-pratico/18533) 😉. So, make sure to add this skill to your set of abilities, as it is not only useful but will improve the quality of your code.